bin/wp-env-config.sh

```diff
@@ -20,4 +20,13 @@ wp plugin list
 wp theme activate storefront
 wp wc customer update 1 --user=1 --billing='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}' --shipping='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}'
 
+## Prepare translation for the test suite
+wp language core install nl_NL
+wp language plugin install woocommerce nl_NL
```

Comment:
I don't think we need to install the language packs for `woocommerce`, `storefront` or `twentytwenty` here. The tests shouldn't be testing core woocommerce or theme functionality, they should only test translations that come from the woocommerce blocks repo.